apps/spruce/src/pages/version/Metadata.tsx

```diff
@@ -231,7 +231,7 @@ const BaseCommitMetadata: React.FC<BaseCommitMetadataProps> = ({
 
   return (
     <MetadataItem>
-      Base commit:{" "}
+      <MetadataLabel>Base commit:</MetadataLabel>{" "}
       {isBaseVersionPending ? (
         <InlineCode data-cy="patch-base-commit">
           {shortenGithash(revision)}
```

Comment:
It might make sense to also remove the label for GitHub merge queue and just show the link since it's a bit repetitive:

<img width="264" alt="Screenshot 2024-08-30 at 1 49 27 PM" src="https://github.com/user-attachments/assets/df7dd4a6-f3ab-4118-b737-af4a987d645d">